tests/python/relax/test_codegen_cutlass.py

```diff
@@ -226,7 +226,7 @@ def get_result_with_relax_cutlass_offload(mod, *args):
 
     seq = tvm.transform.Sequential(
         [
-            relax.transform.FuseOpsByPattern(patterns, annotate_codegen=True),
+            relax.transform.FuseOpsByPattern(patterns, lift_constants=True, annotate_codegen=True),
```

Comment:
I didn't realize that we have `partition_for_cutlass` for Relax, let's use that here.

Reply:
got some errors when using `partition_for_cutlass`, seems there are some issues with pattern registry, will fix and update it in a separate PR